management/appointments/index.php

```diff
@@ -0,0 +1,40 @@
+<?php $root = realpath($_SERVER["DOCUMENT_ROOT"]) ?>
+<!DOCTYPE html>
+<html class="no-js" lang="">
+<head>
+	<title>VITA Appointment Management</title>
+	<?php require_once "$root/server/header.php" ?>
+	<link rel="stylesheet" href="/management/appointments/appointments.css">
+</head>
+<body>
+	<?php
+		$page_subtitle = "Appointment Management";
+		require_once "$root/components/nav.php";
+	?>
+
+	<div class="container">
+		<h2> Download Appointment Schedule for Site </h2>
+		<input type="date" id="dateInput" value="<?php echo date('Y-m-d'); ?>"/>
+		<select id="siteSelect">
+			<?php
+				require_once "$root/server/config.php";
+				GLOBAL $DB_CONN;
+				
+				$query = "SELECT siteId, title FROM Site";
+				$stmt = $DB_CONN->prepare($query);
+
+				$stmt->execute();
+				$sites = $stmt->fetchAll(PDO::FETCH_ASSOC);
```

Comment:
Just from a design standpoint, I'm not a fan of having back-end/database calls in a file that should be for the front-end/view. In this case I think it's probably fine because it's just grabbing the names of the sites, so it would probably be a bit extra to have a separate file.

Reply:
Oh yah I meant to pull this one out, thanks Spencer